src/dashboard/src/main/migrations/0053_quote_args.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def data_migration_put_quotes_in_compress_aip_args(apps, schema_editor):
```

Comment:
Does the function name here really reflect what it's doing? Relatedly, why are you targeting these and only these 6 `StandardTaskConfig`s? They seem to all contain values for `SIPName`. But there are 18 others:

    >>> len(models.StandardTaskConfig.objects.filter(arguments__contains='SIPName'))
    24

Why do the others not need mods to their arguments value also?

Maybe a docstring in this migration func would be helpful.